__tests__/name-format.js

```diff
@@ -30,11 +30,23 @@ test("Name format scss", t => {
     .catch(logError)
 
   function checkResult(result) {
-    t.equal(result.warnings().length, 4, "flags 4 warning")
-    t.is(result.warnings()[0].text, "Expected @function name to match specified pattern (scss/at-function-pattern)", "correct warning text")
-    t.is(result.warnings()[1].text, "Expected @mixin name to match specified pattern (scss/at-mixin-pattern)", "correct warning text")
-    t.is(result.warnings()[2].text, "Expected $ variable name to match specified pattern (scss/dollar-variable-pattern)", "correct warning text")
-    t.is(result.warnings()[3].text, "Expected %-placeholder \"%placeHolder\" to match specified pattern (scss/percent-placeholder-pattern)", "correct warning text")
```

Comment:
Need to look into https://github.com/kristerkari/stylelint-scss/pull/428#issuecomment-573423036 and see whats going on this.